der/src/asn1/ia5_string.rs

```diff
@@ -41,31 +41,13 @@ impl<'a> Ia5StringRef<'a> {
             .map(|inner| Self { inner })
             .map_err(|_| Self::TAG.value_error())
     }
-
-    /// Borrow the string as a `str`.
-    pub fn as_str(&self) -> &'a str {
-        self.inner.as_str()
-    }
-
-    /// Borrow the string as bytes.
-    pub fn as_bytes(&self) -> &'a [u8] {
-        self.inner.as_bytes()
-    }
-
-    /// Get the length of the inner byte slice.
-    pub fn len(&self) -> Length {
-        self.inner.len()
-    }
-
-    /// Is the inner string empty?
-    pub fn is_empty(&self) -> bool {
-        self.inner.is_empty()
-    }
 }
 
-impl AsRef<str> for Ia5StringRef<'_> {
```

Comment:
This is a breaking change. I think it's also generally still good to have an `AsRef` impl for each of these.

Reply:
Ack, I can revert this change. However, I thought that Deref + AsRef<str> implementation for StrSplice should be enough. Is it not?

Reply:
No, deref coercion doesn't apply to trait impls, since the trait needs to be impl'd for the parent type